docs/release-notes/next/GH-2845-accounts-state-gc.md

```diff
@@ -0,0 +1,6 @@
+* Adds garbage collector for removing old account states to free up space on disk.
+  By default the GC is disabled, and does the cleanup every 50000 blocks.
+
+  The cleanup and swap of nodes can not be done atomically.
```

Comment:
What does that mean? Cannot be done atomically?

Reply:
The non atomicity here means that the modification of the aec_account_state table is done in 2 steps:
1.) clearing of the table
2.) writing of the reachable nodes

If, between 1 and 2 the node exits, the node needs resync. (But considering the GC should run once in 3 months, and the critical section takes 2-3 seconds, the risk is low.)

Reply:
How does one detect that the node needs resync? Is this done automatically?

Reply:
Also, is a sync actually required? Wouldn't a local audit suffice? All the blocks are still there ...